src/main/java/pageObjects/DevicePage.java

```diff
@@ -78,16 +76,18 @@ public DevicePage(WebDriver driver) {
 
     @FindBy(id = "device-locations-selectList-romania")
     private WebElement locationsRomaniaListElement;
-    @FindBy(id = "device_dropdown_11")
+    @FindBy(id = "device_dropdown_000")
     private WebElement devicesDropdownButtonInactive;
-    @FindBy(id = "device_assign_11")
+    @FindBy(id = "device_assign_000")
```

Comment:
if we create a device before perfoming actions on it, maybe we could use the name in order to find that device and perform actions on it since all action button ids follow the structure -> ""device_dropdown_" + "nameOfDevice"

Reply:
found another way, in the latest commit